### PR TITLE
Bug 1118853 - Clean up calendar tablist underline r=millermedeiros

### PR DESCRIPTION
**Pull request by lambdabaa**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[gaye Gareth Aye (gaye)](https://github.com/gaye) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=df0f58c63880)

**Comment by try-server-hook** [comment]
[gaye Gareth Aye (gaye)](https://github.com/gaye) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=28f3a46be3db)

File: apps/calendar/js/app.js [hunk 1]
```diff
@@ -229,9 +228,14 @@ module.exports = {
...
-      if (event.target !== today) {
-        AccessibilityHelper.setAriaSelected(event.target, tabs);
+      if (event.target === today) {
+        return;
       }
+
+      Array.prototype.forEach.call(
+        tabs,
+        tab => tab.setAttribute('aria-selected', 'false')
+      );
```
Comment (millermedeiros) [inline apps/calendar/js/app.js]: you need to set `[aria-selected]` on the active tab..

Comment (gaye) [inline apps/calendar/js/app.js]: Can you explain that to me?

Comment (millermedeiros) [inline apps/calendar/js/app.js]: [`[aria-selected]`](http://www.w3.org/TR/wai-aria/states_and_properties#aria-selected) is used by screen readers to specify which `tab` is active.

more info about the screen reader: https://wiki.mozilla.org/Accessibility/Mobile/ScreenReader

Comment (millermedeiros) [inline apps/calendar/js/app.js]: more about tabs: https://yzen.github.io/firefoxos/2014/05/01/firefoxos-accessible-tabs.html

**Comment by try-server-hook** [comment]
[gaye Gareth Aye (gaye)](https://github.com/gaye) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=c430c44eeaf5)

**Comment by try-server-hook** [comment]
[gaye Gareth Aye (gaye)](https://github.com/gaye) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=a4cdf6357051)

**Pull request merged**